content/apps/managing-teammates.md

```diff
@@ -5,32 +5,41 @@ menu:
 title: Managing Teammates
 ---
 
-To add a new teammate to your organization:
+## Invite a teammate
+
+If you're part of an agency team using cloud.gov, you can invite other people at your agency to get cloud.gov accounts:
+
+1. [Send them an invite](https://login.cloud.gov/invitations/new) (this may prompt you to log into your cloud.gov account first).
 
-1. [Send them an invite](https://login.cloud.gov/invitations/new).
 1. Confirm with them that they have received and accepted the invite.
-1. Ensure you are running version >= [6.14.0](https://github.com/cloudfoundry/cli/releases/tag/v6.14.0) of the CLI. If not, you will need to [upgrade](https://docs.cloudfoundry.org/devguide/installcf/install-go-cli.html).
+
+## Give roles to a teammate
+
+If you're an Org Manager or Space Manager, here's how to give roles to your teammates to give them permissions. For details about how cloud.gov org and space roles and permissions work, see [Cloud Foundry roles and permissions](http://docs.cloudfoundry.org/concepts/roles.html#roles).
```

Comment:
Maybe preface with "Once your teammates have cloud.gov accounts set up, they'll need roles which enable them to collaborate with you."